src/Utils/util.test.ts

```diff
@@ -70,4 +71,221 @@ describe('util', () => {
             expect(actual).toBe(true)
         })
     })
+
+    describe('isActionUniqueTEXT', () => {
+        it('exercise TEXT action creation', () => {
+            const sampleAction: CLM.ActionBase = {
+                actionId: "103726c8-2cfe-46e6-ad1e-fef8dbe4d6c3",
+                actionType: CLM.ActionTypes.TEXT,
+                createdDateTime: "2019-02-25T00:00:00.000Z",
+                isTerminal: true,
+                negativeConditions: [],
+                negativeEntities: [],
+                packageCreationId: 0,
+                packageDeletionId: 0,
+                payload: '{"json":{"kind":"value","document":{"kind":"document","data":{},"nodes":[{"kind":"block","type":"line","isVoid":false,"data":{},"nodes":[{"kind":"text","leaves":[{"kind":"leaf","text":"hi","marks":[]}]}]}]}}}"',
+                requiredConditions: [],
+                requiredEntities: [],
+                requiredEntitiesFromPayload: [],
+                suggestedEntity: null,
+                version: 0
+            }
+
+            let actionSet: CLM.ActionBase[] = []
+
+            test('as first action created', () => {
+                expect(isActionUnique(sampleAction, actionSet)).toBe(true)
+            })
+
+            test('as absolutely identical action', () => {
+                actionSet.push(sampleAction)
+                expect(isActionUnique(sampleAction, actionSet)).toBe(false)
+            })
+
+            test('as a similar, but not identical into very similar set', () => {
+                actionSet = [{ ...sampleAction }]
+                let similar = { ...sampleAction, isTerminal: !sampleAction.isTerminal }
+                expect(isActionUnique(similar, actionSet)).toBe(true)
+            })
+
+            test('as a similar, but not identical into very similar set. how smart is the find operator?', () => {
```

Comment:
"how smart is the find operator?"
I didn't notice this comment before. I didn't think this relied on any `find` operations but only `some` and comparing strings.  Anyways remove it.